tests/php/Core/Manifest/fixtures/classmanifest/vendor/silverstripe/modulecbetter/_config/config.yml

```diff
@@ -0,0 +1,4 @@
+---
```

Comment:
Are these various `/Manifest/` files supposed to be in the PR?  They seem odd

Reply:
Yes, we basically create a new module with some basic stuff